src/main/java/persistence/sql/dml/ColumnValues.java

```diff
@@ -19,6 +21,21 @@ public List<ColumnValue> getValues() {
         return new ArrayList<>(columnValues);
     }
 
+    public boolean hasSameSizeAs(ColumnValues<T> columnValues) {
+        return this.columnValues.size() == columnValues.getValues().size();
+    }
```

Comment:
```suggestion
    public boolean hasSameSizeAs(ColumnValues<T> columnValues) {
        return this.size() == columnValues.size();
    }

    private int size() {
        return columnValues.size();
    }
```

`size`를 물어보는 메소드가 있어도 되겠네요.
그렇다면, 내부상태가 `List<ColumnValue>` 에서 다른것으로 변경되더라도 size에 대한 추가 수정은 한곳으로 모여질수 있을것 같아요 🤔

Reply:
좋은 방법인거 같습니다 :) 피드백 감사합니다~